base/Makefile

```diff
@@ -57,15 +57,12 @@ else
 endif
 
 stop: .cluster-config
-	for resource_type in deployment service hpa; do
-	  kubectl get "$${resource_type}/${CHALLENGE_NAME}" >/dev/null 2>&1 && kubectl delete "$${resource_type}/${CHALLENGE_NAME}" || true
-	done
-	kubectl get "configMap/${CHALLENGE_NAME}-pow" >/dev/null 2>&1 && kubectl delete "configMap/${CHALLENGE_NAME}-pow" || true
+	kubectl delete namespace ${CHALLENGE_NAME}
```

Comment:
challenge stop is really nice now :)

Reply:
Do we still delete the load balancer?

Reply:
I think so, that one should be namespaced too, right?

Reply:
okeey :)

Reply:
yea, sadly.. this part of restarting isn't super nice. I think we should just avoid having people do `make stop` then, and treat every case where that's needed as a bug =(